sashimi-webapp/src/logic/documentPackager/markdownProcessor.js

```diff
@@ -0,0 +1,41 @@
+// Imports:
+// Markdown-It module for markdown processing
+import MarkdownIt from 'markdown-it';
+// Code highlighting plugins
+import hljs from 'highlight.js'; // https://highlightjs.org/
+import mh from 'markdown-it-highlightjs';
+// Importing markdown-it-katex plugin for parsing LaTeX mathematical forumla
+import mk from 'markdown-it-katex';
+
+const md = new MarkdownIt({
+  html: true,
+  breaks: true,
+  linkify: true,
+  typographer: true,
+  highlight(str, lang) {
+    if (lang && hljs.getLanguage(lang)) {
+      try {
+        const val = hljs.highlight(lang, str, true).value;
+        return `<pre class="hljs"><code>${val}</code></pre>`;
+      } catch (__) { return ''; }
+    }
+
+    return `<pre class="hljs"><code>${md.utils.escapeHtml(str)}</code></pre>`;
+  }
+});
+
+// Getting markdown-it to use plugins
+// For KaTeX
+md.use(mk);
```

Comment:
consider using a longer name for `mk`. _e.g._ `mdKatex`.
Otherwise, it will be hard to keep track of the plugins when there are many of them in the file.